### PR TITLE
Added Icon

### LINKED ISSUE #2: Add Extension icon
An icon for each extension shows in list of installed extensions and on the VS Code Extension Marketplace. It'd be nice to add one for vscode-lowercase.

If you'd like me to create a quick one and open a PR with it - let me know. :-)

### PR DESCRIPTION
**Pull request by coliff**
Added Icon and bumped version to 1.0.1

Closes #2

### CODE REVIEW

**Comment by ruiquelhas** [comment]
Thanks for the PR.

Can you please move the image file to `static` folder and change the `icon` path? No need for the version bump (I'll do it in a specific commit after this gets merged). Can you revert that change? 

Also, don't you want any kind of attribution in the `README` or something? Feel free to add it, or just point that to me so I can add it later.

**Comment by ruiquelhas** [comment]
According to the [docs](https://code.visualstudio.com/docs/extensions/publish-extension#_advanced-usage), it looks like the icon should be a squared `128px` PNG file. This one seems to have `256px`, so you also need to change that. Or, am I missing something?

**Comment by coliff** [comment]
Hi - I updated  the PR with the requested changes. For attribution, it'd be nice if you could put:
Icon by coliff (https://github.com/coliff/)

Thanks!

**Comment by ruiquelhas** [comment]
You forgot to update the file path. Also, can you please squash the commits into a single one?

**Comment by ruiquelhas** [comment]
If you feel like in the mood, [vscode-uppercase](https://github.com/ruiquelhas/vscode-uppercase) is also missing an icon! :sweat_smile:

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "displayName": "lowercase",
   "description": "Convert selection to lowercase",
   "version": "1.0.0",
+  "icon": "icon.png",
```
Comment (ruiquelhas) [inline package.json:6]: `static/icon.png`

**Comment by ruiquelhas** [comment]
You also need to update `.vscodeignore` to ignore each one (or all) of the existing `gif` files, otherwise `vsce` won't be able to find the `icon`.  

```diff
diff --git a/.vscodeignore b/.vscodeignore
index 052a18f..4e3de33 100644
--- a/.vscodeignore
+++ b/.vscodeignore
@@ -2,7 +2,7 @@
 .vscode-test/**
 coverage/**
 instrumented/**
-static/**
+static/*.gif
 test/**
 .gitignore
 .travis.yml
```

**Review by ruiquelhas — DISMISSED** [review]

**Comment by coliff** [comment]
I'm not too sure about how to squash the commits to one - actually would you mind finishing off this PR with those little changes? If you can, I'll create a 'vscode-uppercase' PR with an icon :-)

**Comment by ruiquelhas** [comment]
In your case, you can always just go back one commit (keeping the changes) and amend the previous one.

```sh
$ git reset HEAD~1
$ git add .
$ git ci --amend -C HEAD
$ git push origin master -f
```

But, I'll pick it up in any case.

**Pull request merged**